concourse/pipelines/linux-image-build.jsonnet

```diff
@@ -4,6 +4,7 @@ local common = import '../templates/common.libsonnet';
 local daisy = import '../templates/daisy.libsonnet';
 local gcp_secret_manager = import '../templates/gcp-secret-manager.libsonnet';
 local lego = import '../templates/lego.libsonnet';
+local sap_test = import '../templates/sap-test.libsonnet';
```

Comment:
i think this is only needed if we are gonna reuse the stuff, otherwise it can just be declared inline